shell/layouts/default.vue

```diff
@@ -194,224 +209,101 @@ export default {
      * Prevent rendering "outlet" until the route changes to avoid re-rendering old route content after its cluster is unloaded
      */
     clusterAndRouteReady() {
-      return this.clusterReady &&
+      return (
+        this.clusterReady &&
         this.clusterId === this.$route?.params?.cluster &&
-        this.currentProduct?.name === getProductFromRoute(this.$route);
-    },
-
-  },
-
-  watch: {
-    counts(a, b) {
-      if ( a !== b ) {
-        this.queueUpdate();
```

Comment:
Removed all watchers that only do `queueUpdate` which just updates the nav tree, as they are now dependencies of the `groups` computed property